environment/environment.go

```diff
@@ -19,30 +20,31 @@ type Environment interface {
 
 // ServiceDetails contains all of the connectivity properties about a given deployed service
 type ServiceDetails struct {
-	RemoteURL  *url.URL
-	LocalURL   *url.URL
+	RemoteURL *url.URL
+	LocalURL  *url.URL
 }
 
 // GetChainlinkClients will return all instantiated Chainlink clients for a given environment
-func GetChainlinkClients(env Environment) ([]client.Chainlink, []*ServiceDetails, error) {
+func GetChainlinkClients(env Environment) ([]client.Chainlink, error) {
 	var clients []client.Chainlink
 
 	sd, err := env.GetAllServiceDetails(ChainlinkWebPort)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 	for _, service := range sd {
 		linkClient, err := client.NewChainlink(&client.ChainlinkConfig{
 			URL:      service.LocalURL.String(),
 			Email:    "notreal@fakeemail.ch",
 			Password: "twochains",
+			RemoteIP: service.RemoteURL.Hostname(),
```

Comment:
This is where I'd say something like:
> The clients need to be unopinionated to how we deploy environments so it needs to be handled separately.

But it makes the tests much more cleaner, so makes sense.